physx-sys/CHANGELOG.md

```diff
@@ -96,7 +97,9 @@
 
 - Ability to not run the default filter shader before the callback.
 
-[Unreleased]: https://github.com/EmbarkStudios/physx-rs/compare/physx-sys-v0.4.11...HEAD
+[Unreleased]: https://github.com/EmbarkStudios/physx-rs/compare/physx-sys-v0.4.13...HEAD
+[0.4.13]: https://github.com/EmbarkStudios/physx-rs/compare/physx-sys-v0.4.12...physx-sys-v0.4.13
+[0.4.11]: https://github.com/EmbarkStudios/physx-rs/compare/physx-sys-v0.4.11...physx-sys-v0.4.12
```

Comment:
0.4.12

Reply:
done.

Not sure how much point there is to maintaining these though. Doesn't seem to happen consistently, as I had to add one extra.

Reply:
I think this can be automated with cargo-release if we set it up?